packages/foam-vscode/src/features/preview-navigation.ts

```diff
@@ -151,6 +151,11 @@ export const markdownItWithRemoveLinkReferences = (
         if (refKey.includes(ALIAS_DIVIDER_CHAR)) {
```

Comment:
the comment from above "Forget about..." should probably be here?
Also, possibly explain why we are doing that